graph/oval_graph.py

```diff
@@ -378,9 +355,9 @@ def change_position(self, positions, nodes_in_rows):
 
         for row in nodes_in_rows:
             for node in nodes_in_rows[row]:
-                if len(
-                        node['label']) > 6 and len(
-                        node['label']) < 40 or continue_move:
+                if len(node['label']) > 6\
+                        and len(node['label']) < 40\
```

Comment:
Using the backslashes is ugly and I think that they're not necessary.
You can silence the PEP8 warning by enclosing the whole condition inside parentheses.

Something like this could work:
```
if (len(node['label']) > 6                                                     
        and len(node['label']) < 40                                            
        or continue_move):                                                     
    print("hi")  
```